weaviate/connection/rest.go

```diff
@@ -55,12 +55,12 @@ func NewConnection(scheme string, host string, httpClient *http.Client, headers
 }
 
 // WaitForWeaviate waits until weaviate is started up and ready
-func (con *Connection) WaitForWeaviate(startup_period int) error {
-	if startup_period < 0 {
-		return errors.New("'startup_period' needs to be an integer larger than zero")
+func (con *Connection) WaitForWeaviate(startupTimeout time.Duration) error {
+	if startupTimeout < 0 {
+		return errors.New("'startupTimeout' needs to be an integer larger than zero")
```

Comment:
> `"'startupTimeout' needs to be an integer larger than zero"`

I don't think this check is necessary anymore, now that the param type is `time.Duration`.

Reply:
I think durations can be negative, so we would still need a check?

Reply:
But the text needs to be adapted

Reply:
Agreed, let's keep the check. I took a look at some of the `time` packages implementations, such as `NewTicker`, and there is a check for positive duration there as well.